src/main/java/teammates/common/util/Config.java

```diff
@@ -95,7 +95,7 @@ private String getGcsBucketname() {
     }
 
     private String getAppUrl() {
-        return props.getProperty("app.url");
+        return Url.trimTrailingSlash(props.getProperty("app.url"));
     }
```

Comment:
Nope, because the `relativeUrl` field is not set-able. It can be appended with params, but can't be set with any other relative URLs once instantiated.